---
layout: page
title: ""
path: "/scampi/"
---

## scAmpi - A versatile pipeline for single-cell RNA-seq analysis from basics to clinics

Anne Bertolini<sup>1</sup>, Michael Prummer<sup>1</sup>, Mustafa Anil Tuncel, Ulrike Menzel, María Lourdes Rosano-González, Jack Kuipers, Daniel Johannes Stekhoven, Tumor Profiler consortium, Niko Beerenwinkel, Franziska Singer<sup>2</sup>

<sup>1</sup>_Equal contribution_<br/>
<sup>2</sup>_Correspondence to: [singer@nexus.ethz.ch](mailto:singer@nexus.ethz.ch)_

### Abstract

Single-cell RNA sequencing (scRNA-seq) has emerged as a powerful technique to decipher tissue composition at the single-cell level and to inform on disease mechanisms, tumor heterogeneity, and the state of the immune microenvironment. Although multiple methods for the computational analysis of scRNA-seq data exist, their application in a clinical setting demands standardized and reproducible workflows, targeted to extract, condense, and display the clinically relevant information. To this end, we designed scAmpi (Single Cell Analysis mRNA pipeline), a workflow that facilitates scRNA-seq analysis from raw read processing to informing on sample composition, clinically relevant gene and pathway alterations, and in silico identification of personalized candidate drug treatments. We demonstrate the value of this workflow for clinical decision making in a molecular tumor board as part of a clinical study.

### Source code availability

[https://github.com/ETH-NEXUS/scAmpi_single_cell_RNA](https://github.com/ETH-NEXUS/scAmpi_single_cell_RNA)

### Publications

- [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.03.25.437054v1.abstract), March 2021.